src/test/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandlerTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.laa.crime.cfecrime.api.*;
+import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.controllers.CfeCrimeController;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RequestHandlerTest {
+
+    private ObjectMapper objMapper = new ObjectMapper();
+    @Test
```

Comment:
Maybe this should just call the RequestHandler rather than setting up Controllers and invoking them. Would be good to DRY (dont repeat yourself) in these tests, as the assessment is mandatory in the input JSON schema, so these tests will need a common setup method for the CfeCrimeRequest object